docs/manual/spec.md

```diff
@@ -170,17 +170,26 @@ Short (< 70 characters) summary of the package.
 
 Used to declare source(s) used to build the package. All sources will
 will be packaged into source rpms.
-Arbitrary number of sources may be declared, for example:
+Arbitrary number of sources may be declared. A numbered source is
```

Comment:
I think we should keep the numbered syntax as the recommended, happy-path style here, and only mention the fact that unnumbered declarations are also possible in a separate paragraph below the examples.

I also don't like mixing the numbered and unnumbered styles in the examples. It makes the syntax look more complicated than it really is.

Reply:
Same, because the unnumbered declarations are less intuitive, since the macros created to reference the sources are always numbered.